src/components/TransferPage.js

```diff
@@ -29,62 +36,68 @@ const TransferPage = (props) => {
-            <SearchProfiles handleToUid={handleToUid} currEmail={props.userData.email}/>
             <br/>
-            <div className="transfer">
-                <form onSubmit={handleSubmit}>
-                    <label>
-                        Amount:
-                        <input
-                            type="text"
-                            value={amount}
-                            placeholder="Enter Transfer Amount"
-                            onChange={(e) => setAmount(e.target.value)}
-                        />
-                    </label>
-                    <label>
-                        Description:
-                        <input
-                            type="text"
-                            value={description}
-                            placeholder="Enter Description"
-                            onChange={(e) => setDescription(e.target.value)}
-                        />
-                    </label>
-                    <label>
-                        Category:
-                        <select value={category} onChange={(e) => setCategory(e.target.value)}>
-                            <option value="FOOD">Food</option>
-                            <option value="TRAVEL">Travel</option>
-                            <option value="PARTY">Party</option>
-                            <option value="PHONE">Phone</option>
-                            <option value="INTERNET">Internet</option>
-                            <option value="RENT">Rent</option>
-                            <option value="SHOPPING">Shopping</option>
-                            <option value="SPORT">Sport</option>
-                            <option value="VIDEO_GAME">Video Game</option>
-                            <option value="TAXI">Taxi</option>
-                        </select>
-                    </label>
-                    <button type="submit">Make Transfer</button>
-                </form>
-                <div className="message">{message ? <p>{message}</p> : null}</div>
-            </div>
+            <NoticeBanner children={success || noticeBanner}/>
+            <Card className="shadow">
+                <Card.Body>
+                    <SearchProfiles handleToUid={handleToUid} currEmail={props.userData.email}/>
+                    <div className="mb-3 mt-md-4">
+                        <h2 className="fw-bold mb-2 text-uppercase ">Make Transfer</h2>
+                        <p className=" mb-5">Please enter your transfer details!</p>
+                        <div className="mb-3">
+                            <Form ref={formRef}>
+                                <Form.Group className="mb-3" controlId="formAmount">
+                                    <Form.Label>Amount</Form.Label>
+                                    <Form.Control onChange={onAmount}
+                                                  type="amount"
+                                                  placeholder="Enter amount" />
```

Comment:
qq: Is it better to have `type="number"`? I didn't see the check of positive numerical value

Reply:
changed